deploy/base/sa.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: pipeline
```

Comment:
Default service account name is 'pipeline' for OpenShift Pipelines ( https://docs.openshift.com/pipelines/1.14/install_config/customizing-configurations-in-the-tektonconfig-cr.html#op-changing-default-service-account_customizing-configurations-in-the-tektonconfig-cr ). 
( For future : This may be different under Konflux - according to @stuartwdouglas its `appstudio-pipeline` )